src/lib/ui/thermostat/ClimatePresets.tsx

```diff
@@ -40,7 +41,19 @@ export function ClimatePresets(props: ClimatePresetsManagement): JSX.Element {
     climatePresetKeySelectedForDeletion,
     setClimatePresetKeySelectedForDeletion,
   ] = useState<ThermostatClimatePreset['climate_preset_key'] | null>(null)
-  const deleteMutation = useDeleteThermostatClimatePreset()
+
+  const { mutate, isError, error, isPending } =
+    useDeleteThermostatClimatePreset()
+
+  const errorMessage = useMemo(() => {
+    if (!isError) return ''
+
+    if (error?.message != null) {
+      return error.message
+    }
+
+    return t.unknownErrorOccured
+  }, [error, isError])
```

Comment:
If this is just the same as `useErrorMessage`, can we move `useErrorMessage` to `src/lib/errors.ts`?